litespi/modules/__init__.py

```diff
@@ -0,0 +1,2 @@
+from litespi.modules.generated_modules import *
+from litespi.modules.modules import *
```

Comment:
I added _modules_ directory where we can store auto-generated classes and add modules manually in the other file.

Reply:
That sounds like a good solution!